src/js/views/clinicians/sidebar/clinician-sidebar_views.js

```diff
@@ -86,6 +86,37 @@ const InfoView = View.extend({
   `,
 });
 
+const WorklistView = View.extend({
+  className: 'clinician-sidebar__worklist',
+  template: hbs`
+    <h3 class="sidebar__heading">
+      {{formatHTMLMessage (intlGet "clinicians.sidebar.clinicianSidebarViews.worklistView.workspaceName") name=workspaceName}}
+    </h3>
+    <button class="button-primary w-100 u-margin--t-8 js-button">
+      {{far "list"}}<span>{{formatHTMLMessage (intlGet "clinicians.sidebar.clinicianSidebarViews.worklistView.worklistBtn") name=clinicianName}}</span>
+    </button>
+    <div class="clinician-sidebar__worklist-info">
+      {{ @intl.clinicians.sidebar.clinicianSidebarViews.worklistView.workspacesInfo }}
+    </div>
+  `,
+  initialize({ clinician, workspace }) {
+    this.clinician = clinician;
+    this.workspace = workspace;
+  },
+  templateContext() {
+    return {
+      clinicianName: this.clinician.get('name'),
+      workspaceName: this.workspace.get('name'),
```

Comment:
I want to determine that if we aren't listing other workspaces that we still want to show the workspace name